src/AccessibilityInsights.Win32/Properties/AssemblyInfo.cs

```diff
@@ -13,7 +13,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("AccessibilityInsights.Win32")]
-[assembly: AssemblyCopyright("Copyright �  2018")]
+[assembly: AssemblyCopyright("Copyright �  2018")]
```

Comment:
Can you take a glance at this copyright symbol changing?

Reply:
Changed back to ©